source/neuropod/backends/torchscript/test/torchscript_test_utils.hh

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2020 The Neuropod Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include "gtest/gtest.h"
+#include "neuropod/neuropod.hh"
+#include "neuropod/backends/torchscript/torch_tensor.hh"
+
+#include <string>
+#include <vector>
+
+#include <stdlib.h>
+
+void test_torchscript_dict_with_union_value_type_model(neuropod::Neuropod &neuropod)
```

Comment:
You can probably just move this function into an anonymous namespace in the `.cc` file